cypress/e2e/advanced.cy.js

```diff
@@ -0,0 +1,71 @@
+/// <reference types='cypress' />
+
+const doClick = require("../support/DoClick");
+
+describe('Book Store app', () => { 
+  const username = 'TestCypress';
+  const book = 'Speaking JavaScript';
+  const idBook= 'book=9781449365035';
+
+  beforeEach(() => {
+
+    cy.logInNewUser().as('user')
+    cy.visit('https://demoqa.com/profile') 
+  });
+
+  //afterEach(() =>{
+    //if (cy.contains((('#see-book-Speaking\ JavaScript > a')))) {
+       //doClick.click('#delete-record-undefined');
+       //doClick.click('#closeSmallModal-ok');
+    //}
+  //})
+ 
+  it('LogIn is done successfully', () => {   
+    cy.get('#userName-value').should('contain.text',username); 
+    cy.url().should('equal', Cypress.config().baseUrl + '/profile');  
+  });
+
+  it('Add book', () => {   
+    doClick.clickleftSideBar(':nth-child(4) > .group-header > .header-wrapper');
```

Comment:
Could you create a method for this action, and re-use it whenever it needed?

Reply:
done